apiary.apib

```diff
@@ -647,6 +647,7 @@ Get a list of contacts.
                 + tags: prospect, expo (array[string])
                 + added_at: `2016-02-04T16:44:33+00:00` (string)
                 + updated_at: `2016-02-05T16:44:33+00:00` (string)
+                + canonical_url: `https://app.teamleader.eu/contact_detail.php?id=2a39e420-3ba3-4384-8024-fa702ef99c9f`
```

Comment:
Q: Should we start using meta data as a root for these? For me this is meta data and does not belong to the model itself.

Reply:
added_at and updated_at is also metadata imo

Reply:
The model can know about its own details, like creation & update times.
It should/can not know its own location in the application.

TBD offcourse :) that is why this is a proposal right? 😉

Reply:
Yes, we kind of need this implemented, but aren't sure yet where to put it. Summoning @stivni and @jenssegers here.

Reply:
@mrtus we can't really differentiate between a `meta` and a `data` object anymore, since in `.list` endpoints, all data is returned as an array in the `data` key.

We could put an array `meta` in the `data` itself, but this doesn't give a lot of advantages imo.